swagger.yaml

```diff
@@ -403,6 +413,29 @@ parameters:
     type: string
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
+  addressType:
+    name: addressType
+    in: query
+    description: |
+      Filter by phone type. Common phone types and their descriptions:
+      - "CM" - Current Mailing
+      - "PA" - Student Alternate Contact
+      - "DM" - Department (US Mail)
+      - "RK" - Financial Aid Address
+      - "RH" - Residence Hall
+    required: false
+    type: string
```

Comment:
And it should be enum. I also noticed I forgot to add it too, can you fix it as well? https://github.com/osu-mist/persons-api/blob/ea4efc6976213d31ba59598cf3661db2e853413b/swagger.yaml#L913-L916